app/push/server/push.js

```diff
@@ -128,21 +133,19 @@ export class PushClass {
 				return;
 			}
 
-			if (response?.statusCode === 422) {
+			if (response.statusCode === 422) {
```

Comment:
Just to be sure: is statusCode guaranteed to be present? (I noticed you removed the ? op)

Reply:
undici types don't seem to indicate it as possibly undefined. but i've wrapped it around a try catch block so this shouldn't be a problem